integr8ly/integreatly-operator
# Update prometheus reconcile rules so not always true

# Description

issue:https://issues.redhat.com/browse/MGDAPI-1017
remove RHOAMInstallationControllerIsNotReconciling and RHOAMInstallationControllerStoppedReconciling as these alerts constantly trigger and replace with RHOAMInstallationControllerIsInReconcilingErrorState

## Type of change



### Verification
- change the operator.yaml image to point at this quay.io/austincunningham/integreatly-operator:v2.7.0 and deploy to redhat-rhmi-operator namespace, use a managed-api cr to deploy RHOAM

or you can look at it deployed here https://console-openshift-console.apps.mw-collab-byoc.sskm.s1.devshift.org/
Verify if  RHOAMInstallationControllerIsNotReconciling and RHOAMInstallationControllerStoppedReconciling are removed and replace with RHOAMInstallationControllerIsInReconcilingErrorState and is present and not firing:

- Find route for Prometheus in redhat-rhmi-middleware-monitoring-operator namespace
- Open its URL
- Go to the Alerts tab and look for RHOAMInstallationControllerIsInReconcilingErrorState
- Click on the expr
- In the query page on the execute button 
- Verify if the rhoam_status metric no data is returned

## Checklist
